cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -445,19 +446,24 @@ def get_user_specific_instrument_list(app):
                 roles = tokenHelper.get_token_roles(decoded_token)
                 email = tokenHelper.get_token_user_email_address(decoded_token)
 
-        out_instrument_list = []
-        for instrument_factory in importer.instrument_factory_iter:
-            if hasattr(instrument_factory, 'instrument_query'):
-                instrument_query = instrument_factory.instrument_query
-                instr_name = getattr(instrument_factory, 'instr_name', instrument_factory().name)
-            else:
-                instrument = instrument_factory()
-                instrument_query = instrument.instrumet_query
-                instr_name = instrument.name
-                
+        with block_timer(logger=logger, 
```

Comment:
no sentry?

Reply:
It's [being set by default](https://github.com/oda-hub/dispatcher-app/blob/d841260bcb3b9208440244facc308d83f93e7180/cdci_data_analysis/timer.py#L7) to the dispatcher's main sentry instance